controls/accordion/accordion-item.md

```diff
@@ -27,11 +27,11 @@ The indicator is the little triangle that is rotated according to whether the Ac
 
 ## IsExpanded
 
-Change the expanded state of the AccordionItem using the `IsExpanded`(`bool`)property.
+Change the expanded state of the `AccordionItem` using the `IsExpanded`(`bool`)property.
```

Comment:
When not part of the code, always use spaces between the words that precede or follow content in parentheses. This is a very common mistake in the MAUI docs.